personalized_nlp/datasets/datamodule_base.py

```diff
@@ -422,37 +423,44 @@ def get_conformity(self, annotations: pd.DataFrame = None) -> pd.DataFrame:
         negative_df = df[df.text_major_vote == 0]
 
         conformity_df = df.groupby("annotator_id").agg(
-            conformity=("is_major_vote", "mean")
-        )
-        conformity_df["pos_conformity"] = positive_df.groupby("annotator_id").agg(
-            pos_conformity=("is_major_vote", "mean")
-        )
-        conformity_df["neg_conformity"] = negative_df.groupby("annotator_id").agg(
-            neg_conformity=("is_major_vote", "mean")
-        )
+            conformity=("is_major_vote", "mean"))
+        conformity_df["pos_conformity"] = positive_df.groupby(
+            "annotator_id").agg(pos_conformity=("is_major_vote", "mean"))
+        conformity_df["neg_conformity"] = negative_df.groupby(
+            "annotator_id").agg(neg_conformity=("is_major_vote", "mean"))
 
         return conformity_df
 
     def limit_past_annotations(self, limit: int):
 
-        past_annotations = self.annotations.merge(self.data[self.data.split == "past"])
+        past_annotations = self.annotations.merge(
+            self.data[self.data.split == "past"])
 
-        text_stds = (
-            past_annotations.groupby("text_id")[self.annotation_column]
-            .agg("std")
-            .reset_index()
-        )
+        text_stds = (past_annotations.groupby("text_id")[
+            self.annotation_column].agg("std").reset_index())
         text_stds.columns = ["text_id", "std"]
 
         past_annotations = past_annotations.merge(text_stds)
 
-        controversial_annotations = past_annotations.groupby("annotator_id").apply(
-            lambda x: x.sort_values(by="std", ascending=False)[:limit]
-        )
+        controversial_annotations = past_annotations.groupby(
+            "annotator_id").apply(
+                lambda x: x.sort_values(by="std", ascending=False)[:limit])
+
+        past_split_text_ids = self.data[self.data.split ==
+                                        "past"].text_id.tolist()
+        non_past_annotations = self.annotations[~self.annotations["text_id"].
+                                                isin(past_split_text_ids)]
+
+        self.annotations = pd.concat(
+            [non_past_annotations, controversial_annotations])
 
-        past_split_text_ids = self.data[self.data.split == "past"].text_id.tolist()
-        non_past_annotations = self.annotations[
-            ~self.annotations["text_id"].isin(past_split_text_ids)
-        ]
+    def compute_average_annotations(self):
+        """Computes average annotations for each annotator"""
```

Comment:
Thank you, previously I wrote another method because I don't want to touch the other codes as much as possible, I don't want to mess them up  ^^;    I will integrate them both